src/context/likedVideos-context.js

```diff
@@ -0,0 +1,57 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios";
+
+const LikedVideosContext = createContext();
+
+const LikedVideosProvider = ({ children }) => {
+  const [likedVideos, setLikedVideos] = useState([]);
+
+  const addToLikedVideos = async (video) => {
+    try {
+      const token = localStorage.getItem("token");
+      const response = await axios.post(
+        "/api/user/likes",
+        { video },
+        {
+          headers: {
+            authorization: token,
+          },
```

Comment:
suggestion: use axios interceptors to add headers in one place for all the request

Reply:
yess